vyper/utils.py

```diff
@@ -172,7 +172,7 @@ def in_bounds(cls, type_str, value):
     '_init_', '___init___', '____init____',
     '_default_', '___default___', '____default____',
     # environment variables
-    'block', 'msg', 'tx',
+    'block', 'msg', 'tx', 'chain', 'chainid',
```

Comment:
is this to block use of `chainid` as a variable?

Reply:
yes `chain.id` and `chainid` are too close IMO